admission_control/admission-control-service/src/admission_control_service.rs

```diff
@@ -112,7 +112,7 @@ where
 
         let txn_proto = req.transaction.clone().unwrap_or_else(Default::default);
 
-        let signed_txn = match SignedTransaction::try_from(txn_proto.clone()) {
+        let transaction = match SignedTransaction::try_from(txn_proto.clone()) {
```

Comment:
I actually think it'll be helpful to keep it `signed_txn` or rename to `signed_user_txn` when it is of that type. Otherwise the word `transaction` is more confusion in our codebase.